### DISCUSSION TITLE
Is there a way to send commands to execute in qutebrowser from terminal?

### DISCUSSION THREAD

**Discussion by fritzrehde** [qutebrowser/qutebrowser#7434] [Q&A] [answered]
I am writing a theme switching shell script and would like to control my one running instance of qutebrowser from the shell script. Is this possible? Can I send "source-config" to qutebrowser so that it executes it somehow?

**Comment by mschilli87** [comment, marked as answer]
```sh
man qutebrowser
```

starts with

```
QUTEBROWSER(1)                          qutebrowser manpage                         QUTEBROWSER(1)

NAME
       qutebrowser - a keyboard-driven, vim-like browser based on PyQt5.

SYNOPSIS
       qutebrowser [-OPTION [...]] [:COMMAND [...]] [URL [...]

[...]
```

so

```sh
qutebrowser ':config-source <config file>'
```

should do what you want AFAICT.

At least

```sh
qutebrowser ':zoom-in'
```

zooms in the current tab of my existing qutebrowser session.

**Reply by fritzrehde** [reply]
Totally missed that in the man page, thanks!
